src/module.rs

```diff
@@ -133,6 +139,63 @@ impl<'d> Iterator for SBModuleSectionIter<'d> {
 
 impl<'d> ExactSizeIterator for SBModuleSectionIter<'d> {}
 
+/// The list of symbols in the module
+pub struct ModuleSymbols<'d> {
```

Comment:
Curious to know why... but not going to block on it.

Reply:
When I want to get first library (excluding exexutable), I think it should be possible to access 1th element without accessing 0th element, but I came up with implementing `nth` of iterator is enough so I'll do this instead of intermediate struct.